### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by LaskoA**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,67 @@
...
+            self.km += other
+        else:
+            self.km += other.km
+        return self
+
+    def __mul__(self, other):
+        if isinstance(other, (int, float)):
+            total = self.km * other
+        else:
+            total = self.km * other.km
```
Comment (IvanRamyk) [inline app/main.py:29]: Distance shouldn't be multiplied by other distance
Please, remove this logic

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

**Comment by LaskoA** [comment]
Please check now after __mul__ method was fixed

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,63 @@
...
+        return Distance(self.km * other)
+
+    def __truediv__(self, other):
+        total = round(self.km / other, 2)
+        return Distance(total)
+
+    def __lt__(self, other):
+        if isinstance(other, (int, float)):
+            return self.km < other
+        else:
```
Comment (IvanRamyk) [inline app/main.py:35]: Redundant `else`
Please fix it in other places too.

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

**Comment by LaskoA** [comment]
Else parts were deleted from lt, le, ge... Please check again :)

**Pull request closed without merging**